phpBB/styles/prosilver/theme/colours.css

```diff
@@ -1151,3 +1151,29 @@ input.disabled {
 	background-color: #d41142;
 	color: #ffffff;
 }
+
+/* pm.css */
+
+.pm-list {
+	background-color: #edf4f7;
+}
+
+.pm-unread .pm-title {
```

Comment:
remove this block

Reply:
yes remove it if we need the color then add the pm-unread class to the A tag...but I do not think we need it.